course/include/isometry.h

```diff
@@ -0,0 +1,73 @@
+#ifndef ISOMETRY_H
+#define ISOMETRY_H
+
+#include "matrix3.h"
+#include "vector3.h"
+
+namespace ekumen {
+namespace math {
+
+// Represents an Homogeneous matrix to perform isometry transformations.
+class Isometry {
+ public:
+  explicit Isometry(const Vector3& translation = Vector3::kZero,
+                    const Matrix3& rotation = Matrix3::kIdentity);
+  explicit Isometry(const Matrix3& rotation);
+  Isometry(const Isometry& obj);
+  Isometry(Isometry&& obj);
+
+  Isometry& operator=(const Isometry& obj);
+  Isometry& operator=(Isometry&& obj);
+
+  // Returns an isometry transformation from a pure translation.
+  static Isometry FromTranslation(const Vector3& translation);
+
+  // Returns an isometry transformation from a pure rotation around an axis.
+  static Isometry RotateAround(const Vector3& axis, const double& angle);
+
+  // Returns an isometry transformation from a pure rotation around Euler angles
+  // (in the x-y-z or pitch-roll-yaw convention).
+  static Isometry FromEulerAngles(const double& psi, const double& theta,
+                                  const double& phi);
+
+  // Gets the rotation matrix.
+  Matrix3 rotation() const;
```

Comment:
Better return `const Matrix3 &` and avoid a copy.

Reply:
Right! Done.